src/types/form.ts

```diff
@@ -197,27 +197,33 @@ export type UseFormRegister<TFieldValues extends FieldValues> = <
   options?: RegisterOptions<TFieldValues, TFieldName>,
 ) => UseFormRegisterReturn;
 
+export type SetFocusOptions = Partial<{
```

Comment:
```suggestion
export type SetFocusConfig = Partial<{
```

Config for consistency.

Reply:
Earlier in this file
```ts
export type UseFormReset<TFieldValues extends FieldValues> = (
  values?: DefaultValues<TFieldValues> | UnpackNestedValue<TFieldValues>,
  keepStateOptions?: KeepStateOptions,
) => void;
```

Reply:
maybe it's time to fix this so we go either Options or Config, happy with either. 🙏

Reply:
I am happy to do the clean-up in another PR. You can ignore this comment. 👍

Reply:
I'd prefer to use options, as it's the naming convention React uses..

Reply:
I will change all to options 👍